pay-api/src/pay_api/resources/fas/routing_slip.py

```diff
@@ -87,6 +88,30 @@ def post():
         return jsonify(response), status
 
 
+@cors_preflight('POST')
+@API.route('/<string:day>/reports', methods=['POST', 'OPTIONS'])
```

Comment:
its the date rite?or just the day as input param?

Reply:
will update it